docs/tutorials/aws/threat-detection.md

```diff
@@ -0,0 +1,23 @@
+# Threat Detection
+
+Prowler allows you to follow a process of Threat Detection in AWS. To run checks related with Threat Detection use:
+```
+prowler aws --category threat-detection
+```
+This comand will run these checks:
+
+* cloudtrail_threat_detection_privilege_escalation
+* cloudtrail_threat_detection_enumeration
+
+*Threat Detection checks will be only executed using `--category threat-detection` flag due to preformance.*
```

Comment:
Add this with the `note` format. See other docs.

Reply:
Done!